app/assets/javascripts/statistics.js

```diff
@@ -64,6 +71,18 @@ var getScale = function getScale() {
   return scale
 }
 
+var prohibitStep = function prohibitStep(elementSelector) {
```

Comment:
@dissident if you assign function to variable you don't need to name it. So:
```js
var prohibitStep = function (..) { ... }
```
or
```js
function prohibitStep(...) {...}
```